test/integ/aio/conftest.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+import pytest
+
+from snowflake.connector.aio import SnowflakeConnection
+
+
+@pytest.fixture()
+def aio_connection(db_parameters):
+    cnx = SnowflakeConnection(
+        user=db_parameters["user"],
+        password=db_parameters["password"],
+        host=db_parameters["host"],
+        port=db_parameters["port"],
+        account=db_parameters["account"],
+        database=db_parameters["database"],
+        schema=db_parameters["schema"],
+        warehouse=db_parameters["warehouse"],
+        protocol=db_parameters["protocol"],
+        timezone="UTC",
+    )
+    yield cnx
+    cnx.close()
```

Comment:
async connection close is awaitable
```suggestion
    await cnx.close()
```